theano/tensor/opt.py

```diff
@@ -4653,6 +4696,10 @@ def same(x, y):
             # This happen with test
             # theano/tensor/tests/test_opt.py:T_local_switch_sink
             new.tag.values_eq_approx = values_eq_approx_remove_inf_nan
+
+            # Julian: Pascal, maybe you can help me implement the copying of the stacktrace for this class?
+            # Because, it's so general I think we need to copy over the stacktraces of all ops being replaced
+            # to every new op?
```

Comment:
I'll have to think about it. It can indeed rewrite large graphs, and I'm not sure what would be the right thing.